openstudiocore/src/energyplus/ReverseTranslator/ReverseTranslateEnergyManagementSystemOutputVariable.cpp

```diff
@@ -0,0 +1,136 @@
+/**********************************************************************
+ *  Copyright (c) 2008-2016, Alliance for Sustainable Energy.
+ *  All rights reserved.
+ *
+ *  This library is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU Lesser General Public
+ *  License as published by the Free Software Foundation; either
+ *  version 2.1 of the License, or (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *  Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public
+ *  License along with this library; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ **********************************************************************/
+
+#include "../ReverseTranslator.hpp"
+
+#include "../../model/EnergyManagementSystemOutputVariable.hpp"
+#include "../../model/EnergyManagementSystemOutputVariable_Impl.hpp"
+#include "../../model/EnergyManagementSystemProgram.hpp"
+#include "../../model/EnergyManagementSystemProgram_Impl.hpp"
+#include "../../model/EnergyManagementSystemSubroutine.hpp"
+#include "../../model/EnergyManagementSystemSubroutine_Impl.hpp"
+#include "../../model/EnergyManagementSystemGlobalVariable.hpp"
+#include "../../model/EnergyManagementSystemGlobalVariable_Impl.hpp"
+
+#include <utilities/idd/EnergyManagementSystem_OutputVariable_FieldEnums.hxx>
+#include "../../utilities/idd/IddEnums.hpp"
+#include <utilities/idd/IddEnums.hxx>
+
+using namespace openstudio::model;
+
+namespace openstudio {
+
+namespace energyplus {
+
+OptionalModelObject ReverseTranslator::translateEnergyManagementSystemOutputVariable(const WorkspaceObject & workspaceObject)
```

Comment:
@brianlball it would be good to test this by importing E+ example files with EMS objects, did you try that?